tests/test_backend_qibotn.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from qibotn.backends import CuTensorNet, QuimbBackend
```

Comment:
Importing `CuTensorNet` will fail the CI test because the GitHub runner doesn't have `cuquantum` for GPUs.

@alecandido Shall we take it out and only test `QuimbBackend` for now? We can add it back once we have self-hosted GPU machines ready as the runners.

Reply:
Yes, I think so.

Reply:
We tried a few ways to remove `cuquantum`, but even only importing `quimb` will inevitably trigger importing of `cuquantum`. So it's a bit tricky to test `quimb` without involving `cuquantum`.

Reply:
I have remove the import from the test in this PR, however in https://github.com/qiboteam/qibotn/blob/main/src/qibotn/backends/__init__.py we are importing cuquantum. We should change that in the qibotn repo first.

Reply:
Yeah, indeed. It's doable, but after taking `cuquantum` out from `__init__.py`, there're some files that need to be modified to import `cuquantum` in a not-so-convenient way.

Reply:
As @Tankya2 pointed out, this `test_backend_qibotn.py` file works if there's a GPU runner. Does it make sense if we use a self-hosted GPU runner for this?

Reply:
To catch up with the release, could we temporarily skip the test of qibotn for now, and once the GPU runners are set up, we enable the test again?

Reply:
As you can see, after my commits, tests are not crashing due to cuquantum. However quimb and qibojit are interfering with each other through numba. Most likely @Tankya2 and @Vinitha-balachandran have already faced this problem in the past...

Reply:
@Tankya2 and I didn't see this before.
@Vinitha-balachandran suggested setting the backend to numpy only for quimb test. I noticed the backends to be tested on are defined in `conftest.py` below.
```py
# backends to be tested
BACKENDS = ["numpy", "tensorflow", "qibojit-numba", "qibojit-cupy", "qibojit-cuquantum"]
```
Is it possible to only select `numpy` backend for the quimb test for now, so as to avoid the involvement with `qibojit-numba`?

Reply:
@liweintu this is now fixed (see my last commit). We can know test quimb with numba.

Reply:
@scarrazza That's great! Thanks a lot for the help on the fix.